### ISSUE TITLE
travel expenses section does not take into account freelancers / unemployed people

### ISSUE THREAD

**Issue by xrmx** [kosamari/who-pays-tech-speakers#5]
As the sentence starts with "my employer" in some cases it may not be applicable.

**Comment by bookercodes** [comment]
I _think_ you are supposed to leave the field blank if it does not apply to you.

**Comment by kosamari** [comment]
@xrmx thanks ! you can leave the field empty as @alexbooker mentioned. That specific question is in there since a lot of conference offers "we'll list your company as sponsor is they pay for your travel".

maybe make the selection to [my employer / my business] ?  If you think of any other wording fitting, let me know !

**Comment by cczona** [comment]
That'd still leave a dilemma for (or under-/mis-representation of), for instance, students and unemployed people.

How about adding:

My own company
My school
My personal funds

**Comment by bookercodes** [comment]
Sounds good, @cczona !

**Comment by cczona** [comment]
Clarifying:

Currently it's

My employer  [e.g. covered] my travel expenses. 
I was  [e.g. still a student] while attending the event.

I suggest:
- flipping those two sentences
- removing "still"
- Conference| covered (all | most | part | none) of my travel travel and lodging expenses. The remainder, if any, was covered by my (Personal funds| Employer | Self-employment company | School | 3rd party sponsor)

**Comment by kosamari** [comment]
Thank you all for the input in this ! I'm making few changes. 
I'd like to keep the data structure the same (not adding new field) regards to this for now, as this change will affect meaning of previous 195 answers. 

Here is new edit ... 
- "I was ____ while attending conference". to be separated from "My employer ____" sentence
- Adding note on employer travel assistance section  "if you had someone else support your travel, We'd love to know about it, comment in Also: section if you like."

I went with "let me know in comment" approach, many conference I spoke offer "If your company can pay your travel, we'll list you as sponsor of conf" if there are alternative way people are finding their way in, I'd like to know more what kind of assistance is.

I did not include "how much I (speaker) paid to speak" aspect to this form by design to keep things simper. When its personal, things gets even more subjective & question like _do I include subway ride to venue in this?_ _I also paid $ for the research for my talk_ etc. (related : question like how much time it took to prep for talk which is very subjective is getting some discussion like #12 & [triggering more ideas](https://twitter.com/KatieK2/status/659499859615965185), also things that are not covered in this form like conference you declined to speak is getting mention #16   )
Perhaps these whopays reports should be starting point for other conversations & site like whopays rather than making one form fits all.

Also the word 'still' is being removed from student selection.
